### ISSUE TITLE
Enable Zend opcache

### ISSUE THREAD

**Issue by danb35** [stephdl/dev#12]
The php-scl installation installs the Zend opcache extension, but does not activate it in php.ini.  A template fragment at `/etc/e-smith/templates/etc/opt/remi/php72/php.ini/75ExtensionSettings01Opcache` with contents of:
```
zend_extension=/opt/rh/rh-php72/root/usr/lib64/php/modules/opcache.so
```
Enables it (for PHP 7.2, of course).  This has the potential to accelerate PHP applications; I don't see any obvious reason it shouldn't be enabled by default.

**Comment by stephdl** [comment]
Thanks for reporting it, does it is valid also for php73 ?

**Comment by danb35** [comment]
I don't see why it shouldn't, but these are the only `opcache.so` files on my system:
```
[root@neth ~]# locate opcache.so
/opt/rh/rh-php56/root/usr/lib64/php/modules/opcache.so
/opt/rh/rh-php71/root/usr/lib64/php/modules/opcache.so
/opt/rh/rh-php72/root/usr/lib64/php/modules/opcache.so
/usr/lib64/php/modules/opcache.so
```
It looks like there's a separate `php73-php-opcache` rpm that isn't installed.

**Comment by stephdl** [comment]
hum, does on php73 it is used another cache accelerator, can we find something with the phpinfo for php73

**Comment by danb35** [comment]
I'd think the more straightforward way to accomplish this would be to pull
in php73-php-opcache from remi's repo (see
https://pkgs.org/download/php73-php-opcache).  A quick web search pulls up
a bunch of very old pages, alternative accelerators that "now work with PHP
5.4!", etc.--I'm not seeing alternatives that are compatible with PHP 7.3.
I don't know if that's also going to configure that scl's php.ini or
related files to load that extension, or if there would need to be a
template fragment to do that.

**Comment by stephdl** [comment]
ok if you look after your locate command, it comes from rh-php* centos scl not from remi and therefore not from my rpm nethserver-php-scl

By the way the recommended setting are https://www.php.net/manual/en/opcache.installation.php#opcache.installation.recommended

obviously we need to enable also with the zend_extension=....

You think we need to enable it in my rpm ?

**Comment by stephdl** [comment]
```
[root@ns7loc12 ~]# locate opcache.so
/opt/remi/php56/root/usr/lib64/php/modules/opcache.so
/opt/remi/php70/root/usr/lib64/php/modules/opcache.so
/opt/remi/php71/root/usr/lib64/php/modules/opcache.so
/opt/remi/php72/root/usr/lib64/php/modules/opcache.so
/opt/remi/php73/root/usr/lib64/php/modules/opcache.so
```

**Comment by stephdl** [comment]
it seems if I install the remi relevant rpms I install and enable opcache 

`yum install  php{71,72,73}-php-opcache`

then check 

```
vim /etc/opt/remi/php73/php.d/10-opcache.ini
vim /etc/opt/remi/php72/php.d/10-opcache.ini
vim /etc/opt/remi/php71/php.d/10-opcache.ini
```

**Comment by stephdl** [comment]
please @danb35 could you test my rpm, it enables opcache for php73.php72,php71

yum install http://mirror.de-labrusse.fr/NethDev/nethserver-php-scl/nethserver-php-scl-1.2.5-1.ns7.x86_64.rpm

in order to test, either drop  a phpinfo inside your virtualhost and change the php version
or in your terminal

scl enable php71 bash
php -r "phpinfo();" |less

search with OPcache and opcache (/ to search with less, use n and N to browse)

then exit and do the same for php72 and php73

check also in /var/log/messages if you have no errors

**Comment by danb35** [comment]
I must be missing a repo--when I try that, even after installing the stephdl repo (and adding `--enablerepo=stephdl`), I get a bunch of dependency errors like:
```
--> Finished Dependency Resolution
Error: Package: nethserver-php-scl-1.2.5-1.ns7.x86_64 (/nethserver-php-scl-1.2.5-1.ns7.x86_64)
           Requires: php72-php-bcmath
Error: Package: nethserver-php-scl-1.2.5-1.ns7.x86_64 (/nethserver-php-scl-1.2.5-1.ns7.x86_64)
           Requires: php71-php-imap
Error: Package: nethserver-php-scl-1.2.5-1.ns7.x86_64 (/nethserver-php-scl-1.2.5-1.ns7.x86_64)
           Requires: php71-php-fpm
Error: Package: nethserver-php-scl-1.2.5-1.ns7.x86_64 (/nethserver-php-scl-1.2.5-1.ns7.x86_64)
           Requires: php71-php-mcrypt
Error: Package: nethserver-php-scl-1.2.5-1.ns7.x86_64 (/nethserver-php-scl-1.2.5-1.ns7.x86_64)
           Requires: php70-php-ldap
Error: Package: nethserver-php-scl-1.2.5-1.ns7.x86_64 (/nethserver-php-scl-1.2.5-1.ns7.x86_64)
           Requires: php72-php-pgsql
Error: Package: nethserver-php-scl-1.2.5-1.ns7.x86_64 (/nethserver-php-scl-1.2.5-1.ns7.x86_64)
           Requires: php73-php-pdo
Error: Package: nethserver-php-scl-1.2.5-1.ns7.x86_64 (/nethserver-php-scl-1.2.5-1.ns7.x86_64)
           Requires: php71-php-bcmath
```

**Comment by stephdl** [comment]
```
  yum install http://rpms.famillecollet.com/enterprise/remi-release-7.rpm
  yum install http://mirror.de-labrusse.fr/NethServer/7/x86_64/nethserver-stephdl-1.0.7-1.ns7.sdl.noarch.rpm
 yum install http://mirror.de-labrusse.fr/NethDev/nethserver-php-scl/nethserver-php-scl-1.2.5-1.ns7.x86_64.rpm --enablerepo=stephdl
```

**Comment by danb35** [comment]
OK, with those installed, and the root PHP version set to 7.1, 7.2, and 7.3, all show OPCache enabled on the phpinfo page.

**Comment by stephdl** [comment]
verified, released as nethserver-php-scl-1.2.5-1.ns7.sdl.x86_64.rpm
